src/main/java/cart/domain/Grade.java

```diff
@@ -0,0 +1,55 @@
+package cart.domain;
+
+public enum Grade {
+
+    GENERAL("일반",0),
+
+    SILVER("silver",5),
+    GOLD("gold",10),
+    PLATINUM("platinum",15),
+    DIAMOND("diamond", 20);
+
+    private String gradeName;
+    private int gradeDiscount;
+
+    Grade(String gradeName, int gradeDiscount) {
+        this.gradeName = gradeName;
+        this.gradeDiscount = gradeDiscount;
+    }
+
+    public String getGradeName() {
+        return gradeName;
+    }
+
+    public int getGradeDiscount() {
+        return gradeDiscount;
+    }
+    
+    public static int findGradeDiscount(String grade){
+        if (grade.equals(Grade.GENERAL.gradeName)) {
+            return 0;
```

Comment:
```suggestion
            return Grade.GENERAL.gradeDiscount;
```

이렇게 하면 나중에 Grade의 요구사항이 달라져도 리팩토링 할 부분이 줄어들 것 같습니다 🙂

Reply:
헉 그렇네요!!! 춘식 대박... 감사합니다!